docs/site/migration/models/methods.md

```diff
@@ -6,8 +6,309 @@ sidebar: lb4_sidebar
+  }
+
+  @get('/notes/count', {
+    responses: {
+      '200': {
+        description: 'Note model count',
+        content: {'application/json': {schema: CountSchema}},
+      },
+    },
+  })
+  async count(
+    @param.query.object('where', getWhereSchemaFor(Note)) where?: Where<Note>,
+  ): Promise<Count> {
+    return this.noteRepository.count(where);
+  }
+
+  @get('/notes', {
+    responses: {
+      '200': {
+        description: 'Array of Note model instances',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'array',
+              items: getModelSchemaRef(Note, {includeRelations: true}),
+            },
+          },
+        },
+      },
+    },
+  })
+  async find(
+    @param.query.object('filter', getFilterSchemaFor(Note)) filter?: Filter<Note>,
+  ): Promise<Note[]> {
+    return this.noteRepository.find(filter);
+  }
+
+  @patch('/notes', {
+    responses: {
+      '200': {
+        description: 'Note PATCH success count',
+        content: {'application/json': {schema: CountSchema}},
+      },
+    },
+  })
+  async updateAll(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(Note, {partial: true}),
+        },
+      },
+    })
+    note: Note,
+    @param.query.object('where', getWhereSchemaFor(Note)) where?: Where<Note>,
+  ): Promise<Count> {
+    return this.noteRepository.updateAll(note, where);
+  }
+
+  @get('/notes/{id}', {
+    responses: {
+      '200': {
+        description: 'Note model instance',
+        content: {
+          'application/json': {
+            schema: getModelSchemaRef(Note, {includeRelations: true}),
+          },
+        },
+      },
+    },
+  })
+  async findById(
+    @param.path.number('id') id: number,
+    @param.query.object('filter', getFilterSchemaFor(Note)) filter?: Filter<Note>
+  ): Promise<Note> {
+    return this.noteRepository.findById(id, filter);
+  }
+
+  @patch('/notes/{id}', {
+    responses: {
+      '204': {
+        description: 'Note PATCH success',
+      },
+    },
+  })
+  async updateById(
+    @param.path.number('id') id: number,
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(Note, {partial: true}),
+        },
+      },
+    })
+    note: Note,
+  ): Promise<void> {
+    await this.noteRepository.updateById(id, note);
+  }
+
+  @put('/notes/{id}', {
+    responses: {
+      '204': {
+        description: 'Note PUT success',
+      },
+    },
+  })
+  async replaceById(
+    @param.path.number('id') id: number,
+    @requestBody() note: Note,
+  ): Promise<void> {
+    await this.noteRepository.replaceById(id, note);
+  }
+
+  @del('/notes/{id}', {
+    responses: {
+      '204': {
+        description: 'Note DELETE success',
+      },
+    },
+  })
+  async deleteById(@param.path.number('id') id: number): Promise<void> {
+    await this.noteRepository.deleteById(id);
+  }
```

Comment:
I find this code snippet as way too long, hiding more important content on this page.

I am also concerned about maintenance costs, I am pretty much certain we will often forget to update this code after modifying controller templates in our CLI.

Can you please shorten this snippet and show only one or two methods? You can point users to one of our example controllers (e.g. https://github.com/strongloop/loopback-next/blob/master/examples/todo-list/src/controllers/todo-list.controller.ts) for a full version.